classes/class-wc-connect-options.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Options' ) ) {
+	class WC_Connect_Options {
+		public static $tos_accepted = 'wc_connect_tos_accepted';
```

Comment:
What about using [constants](http://php.net/manual/en/language.oop5.constants.php) instead of static variables? The behaviour is pretty much the same but the intention is clearer.